_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,473 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.field-group {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .field-group-hd {
+        margin-bottom: spacing-vertical(small);
+        font-size: font-size(mid-large);
+    }
+
+    .field {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $field-tip-color;
+        }
+    }
+
+    &.form-group {
```

Comment:
Please see [my note about this class/element](https://github.com/edx/ux-pattern-library/pull/75/files#r32429273) elsewhere.